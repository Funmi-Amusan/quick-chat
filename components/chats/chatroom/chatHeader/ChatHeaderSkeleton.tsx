import { View } from 'react-native';

const ChatHeaderSkeleton = () => {
  return (
    <View className="flex-row items-center">
      <View className="mr-3 h-10 w-10 rounded-full bg-gray-200" />
      <View>
        <View className="h-5 w-32 rounded bg-gray-200" />
        <View className="mt-0.5 h-3 w-20 rounded bg-gray-200" />
      </View>
    </View>
  );
};
export default ChatHeaderSkeleton;