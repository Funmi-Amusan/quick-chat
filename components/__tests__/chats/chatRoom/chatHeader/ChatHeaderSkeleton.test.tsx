import { render } from '@testing-library/react-native';

import ChatHeaderSkeleton from '~/components/chats/chatRoom/chatHeader/ChatHeaderSkeleton';

test('displays chatList item correctly', async () => {
  const replyPreviewComponent = render(<ChatHeaderSkeleton />);

  expect(replyPreviewComponent.toJSON()).toMatchSnapshot();
});
