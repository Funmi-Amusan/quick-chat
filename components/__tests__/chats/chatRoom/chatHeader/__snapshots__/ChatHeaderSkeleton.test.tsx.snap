// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays chatList item correctly 1`] = `
<View
  className="flex-row items-center"
  testID="chat-header-skeleton"
>
  <View
    className="mr-3 h-10 w-10 rounded-full bg-gray-200"
  />
  <View>
    <View
      className="h-5 w-32 rounded bg-gray-200"
    />
    <View
      className="mt-0.5 h-3 w-20 rounded bg-gray-200"
    />
  </View>
</View>
`;
